pythran/pythonic/numpy/abs.hpp

```diff
@@ -2,11 +2,17 @@
 #define PYTHONIC_NUMPY_ABS_HPP
 
 #include "pythonic/utils/proxy.hpp"
-#include"pythonic/types/ndarray.hpp"
+#include "pythonic/types/ndarray.hpp"
 #include "pythonic/types/numexpr_to_ndarray.hpp"
 #include "pythonic/utils/numpy_traits.hpp"
+
 #include <nt2/include/functions/abs.hpp>
 
+// nt2 does not handle bool specialization, which is indeed a corner case...
+namespace nt2 {
+    constexpr bool abs(bool v) { return v;}
+}
+
```

Comment:
It is not a fix.
I missmatch between branches. I was rebased on your "warning free" branch which introduced this patch in isclose.hpp
We should just wait for your branch to be merge then I will rebase on it and all will be fine :-)